homeworks/11/Domashno/src/main/java/com/example/domashno/Coments.java

```diff
@@ -0,0 +1,66 @@
+package com.example.domashno;
+
+import jakarta.servlet.*;
+import jakarta.servlet.http.*;
+import jakarta.servlet.annotation.*;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+
+@WebServlet(name = "Coments", value = "/Coments/*")
+public class Coments extends HttpServlet {

```

Comment:
Methods, concerning the comments service, like put and delete are not defined in this class. I would suggest uniting this class with Delete.java as they refer to different methods of the same service